src/controller.py

```diff
@@ -0,0 +1,77 @@
+import pandas as pd
+from cement import Controller, ex
+
+from .plot.base import Portfolio
+from .utils import VERSION
+
+
+class BaseController(Controller):
+    class Meta:
+        label = 'tw'
+
+        # text displayed at the top of --help output
+        description = 'An easy-to-use command line interface for Tastyworks!'
+
+        # text displayed at the bottom of --help output
+        epilog = f'tastyworks-cli {VERSION}'
+
+        # controller level arguments. ex: 'tw --version'
+        arguments = [
+            # add a version banner
+            (
+                ['-v', '--version'],
+                {
+                    'action': 'version',
+                    'version': VERSION
+                }
+            ),
+        ]
+
+    def _default(self):
+        """Default action if no sub-command is passed."""
+        self.app.args.print_help()
+
+    @ex(
+        help='chart your portfolio\'s net liquidity or realized profit/loss over time',
+        arguments=[
+            (
+                ['csv'],
+                {
+                    'action': 'store',
+                    'help': 'csv file containing full portfolio transaction history'
```

Comment:
I assume this is saying that the `csv` arg should be a path to the csv file, I recommend rewording the help text to specify that this should be a filepath string.

Reply:
also see my comment on the readme, it wouldn't take very much to shed a little bit more light on the nature of the csv file